backend/internal/usecases/create_post_usecase.go

```diff
@@ -0,0 +1,29 @@
+package usecases
+
+import (
+	"myapp/internal/controllers/schema"
+	"myapp/internal/entities"
+	"myapp/internal/interfaces"
+)
+
+type CreatePostUsecase struct {
+	postRepository interfaces.PostRepository
+}
+
+func NewCreatePostUsecase(
+	postRepository interfaces.PostRepository,
+) *CreatePostUsecase {
+	return &CreatePostUsecase{
+		postRepository: postRepository,
+	}
+}
+
+func (u *CreatePostUsecase) Execute(request schema.CreatePostRequest, userId int64) ([]*entities.Post, error) {
+	post, err := u.postRepository.CreatePost(request.Title, request.Body, userId)
```

Comment:
repository の結果をハンドリングすることがなさそうやから、そのまま return していいかも
```suggestion
	return u.postRepository.CreatePost(request.Title, request.Body, userId)
```

Reply:
おっしゃる通りでマジで何もしてないコードだったね...
修正しました！